.github/workflows/test.yml

```diff
@@ -18,9 +18,12 @@ jobs:
       - uses: DeLaGuardo/setup-clojure@master
         with:
           tools-deps: latest
+          lein: latest
       - uses: actions/cache@v2
         with:
           path: ~/.m2
           key: test-m2-${{ hashFiles('deps.edn') }}-v1
+      - name: lint integration-test.sh
+        run: shellcheck integration-testing/test.sh
```

Comment:
It may be good to move this step to `lint.yml`.

BTW, `shellcheck` doesn't required to be set up to work in GitHub Actions?

Reply:
> BTW, shellcheck doesn't required to be set up to work in GitHub Actions?

It's already installed, see https://github.com/koalaman/shellcheck/tree/163b2f12e2076a643e744774c8408370655bfc09#in-your-build-or-test-suites